src/rocksdb/src/lib.rs

```diff
@@ -173,18 +174,22 @@ pub struct RocksDBMetrics {
     /// Latency of multi_gets, in fractional seconds.
     pub multi_get_latency: DeleteOnDropHistogram<'static, Vec<String>>,
     /// Size of multi_get batches.
-    pub multi_get_size: DeleteOnDropHistogram<'static, Vec<String>>,
+    pub multi_get_size: DeleteOnDropCounter<'static, AtomicU64, Vec<String>>,
+    /// Size of multi_get non-empty results.
+    pub multi_get_result_size: DeleteOnDropCounter<'static, AtomicU64, Vec<String>>,
     /// Latency of write batch writes, in fractional seconds.
     pub multi_put_latency: DeleteOnDropHistogram<'static, Vec<String>>,
     /// Size of write batches.
-    pub multi_put_size: DeleteOnDropHistogram<'static, Vec<String>>,
+    pub multi_put_size: DeleteOnDropCounter<'static, AtomicU64, Vec<String>>,
```

Comment:
this will need to be split into 2 parts, a shared and non-shared, where histograms are shared and the others have `worker_id`

Also, we probably need a `TODO` to consider just adding `worker_id` to the histograms anyways

Reply:
> Also, we probably need a TODO to consider just adding worker_id to the histograms anyways

Actually let me see if that would be better. Because then everything is worker specific and we won't need the shared thing anymore